install.sh

```diff
@@ -64,9 +69,26 @@ install_cli() {
     echo "Extracting the OpsLevel CLI..."
     tar -xzf "$TMP_DIR/opslevel.tar.gz" -C "$TMP_DIR"
 
-    echo "Installing the OpsLevel CLI to /usr/local/bin ..."
-    sudo mv "$TMP_DIR/opslevel" /usr/local/bin/
 
+
+    # Search for a writable directory in the PATH
+    TARGET_DIR=""
+    for dir in $(echo "$PATH" | tr ':' '\n'); do
+        if is_writable "$dir"; then
+            TARGET_DIR="$dir"
+            break
+        fi
+    done
+
+    # If no writable directory is found, exit
+    if [ -z "$TARGET_DIR" ]; then
+        echo "Installation failed.  User has no permissions to any directory on PATH"
```

Comment:
```suggestion
    if [ -z "$TARGET_DIR" ]; then
        if [ -n "${GITHUB_PATH:-}" ] && [ -w "$GITHUB_PATH" ]; then
            TARGET_DIR="$HOME/.local/bin"
            mkdir -p "$TARGET_DIR"
            echo "$TARGET_DIR" >> "$GITHUB_PATH"
            echo "No writable directory in PATH. Using GITHUB_PATH to add $TARGET_DIR to PATH."
        else
            echo "Installation failed.  User has no permissions to any directory on PATH"
        fi
```

Reply:
Going to actually move this into the action so it ensures a writable directory is on the path